src/Operations.jl

```diff
@@ -1100,7 +1100,7 @@ end
 function with_temp_env(fn::Function, temp_env::String)
     cache = Base.active_project()
     try
-        Pkg.API.activate(temp_env)
+        Pkg.API.activate(temp_env; silent=true)
```

Comment:
Wouldn't it be better to just `Base.ACTIVE_PROJECT[] = temp_env` instead of introducing a `silent` option?

Reply:
That should have the same effect 👍